src/modules/Crud.test.js

```diff
@@ -0,0 +1,26 @@
+const { addTask, removeTodo } = require('./Crud');
+
+const arrTest = [];
+
+describe('Task', () => {
+  test('adds task to local storage', () => {
+    addTask('test input value', arrTest);
+    const ul = document.createElement('ul');
+    ul.innerHTML += `
+      <li class="task-item" id=${arrTest[0].index}>
+      </li>
+        `;
+    arrTest.push(localStorage.getItem('tasks'));
+    expect(localStorage.getItem('tasks')).not.toBe([]);
+    expect(ul).not.toBeNull();
+  });
+  test('remove task from the local storage and li from the DOM', () => {
+    const todoId = 0;
+    const listContainer = document.createElement('li');
+    listContainer.className = 'task-item';
+    listContainer.id = 0;
+    removeTodo(todoId, listContainer, arrTest);
+    expect(localStorage.getItem('tasks')).toEqual('[]');
+    expect(document.contains(listContainer)).toBe(false);
+  });
+});
```

Comment:
- [ ] According to the project [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_2.md), you are supposed to add test for the following, 
   - a function for editing the task description.
   - a function for updating an item's 'completed' status.
   - the "Clear all completed" function.

Please kindly add them.

Reply:
Hey @t-yanick, Thank you so much for telling us that. Yes, we were supposed to submit this pull request for the first project. Can we still submit it for the first project?